### PR TITLE
[Live-Site] Feat/event logs api

### LINKED ISSUE #706: [Live-Site] Integrate API for event logs
### Requirement
We need to add the API for logs in `live-site`. This API will be polled at some interval and get the list of logs of type `EVENTS_REMOVE_PEER`. Then it will show the API data in UI.

### Acceptance
- Fallback UI should be shown if no data is present.
- It should show logs of only active running event.

### Note
Backend changes will be needed to add event id field in logs.

### PR DESCRIPTION
**Pull request by satyam73**
#####  Closes #706 

Date: 31-10-23

Developer Name: Satyam Bajpai

PR Number(s):- #723

Issue Ticket Number:-  #706 

Backend changes
- [x] Yes
- [ ] No

Frontend Changes
- [x] Yes
- [ ] No

Is Under Feature Flag 
- [x] Yes
- [ ] No

Database changes
- [x] Yes
- [ ] No

Breaking changes (If your feature is breaking/missing something please mention pending tickets)
- [ ] Yes
- [x] No

Deployment notes
Any special instructions for the deployment(N/A)

Tested in staging
- [ ] Yes
- [x] No

Related backend PR
- https://github.com/Real-Dev-Squad/website-backend/pull/1649

### What is the change?
In this PR I have integrated API for event kickout logs and rendered the data in ui.

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots

https://github.com/Real-Dev-Squad/website-www/assets/85296770/72cf3dab-385c-472e-abd2-117ad3f69550

### Test Snapshots
image

**Note :**
What to delete and what not to

- Keep the headings and answer the questions.
- Delete the examples
- Delete the description below headings `What is the change?` & `Before/After Screenshots`

### CODE REVIEW

**Comment by shubhamsigdar1** [comment]
Can you please add screenshot or video in description

File: app/components/events/logs-page.js [hunk 1]
```diff
@@ -1,85 +1,76 @@
...
+    })();
+
+    this.logsPollingInterval = setInterval(async () => {
+      await this.getLogs();
+    }, EVENTS_LOGS_POLL_TIME);
+
+    registerDestructor(this, () => {
+      clearInterval(this.logsPollingInterval);
+    });
+  }
```
Comment (shubhamsigdar1) [inline app/components/events/logs-page.js:37]: What you trying to achieve above from line 30-37

**Review by vinit717 — CHANGES_REQUESTED** [review]
Please improve PR description and why there are no tests

**Review by iamitprakash — CHANGES_REQUESTED** [review]
PR details are missing, please add it

**Review by shubhamsigdar1 — APPROVED** [review]

**Review by vinit717 — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**